components/brave_rewards/core/wallet_provider/uphold/connect_uphold_wallet.h

```diff
@@ -39,11 +39,13 @@ class ConnectUpholdWallet : public wallet_provider::ConnectExternalWallet {
 
   void OnGetCapabilities(ConnectExternalWalletCallback,
                          const std::string& access_token,
+                         const std::string& country_id,
```

Comment:
This can be an rvalue ref.

Reply:
I'll leave the signatures as-is in this PR to avoid mixing rvalue and lvalue refs.